### PR TITLE
KEYCLOAK-14758 reorganized Getting Started files for reuse 

### PR DESCRIPTION
**Pull request by andymunro**
@lhanusov, @pskopek  I made some changes related to the need to make the topics in this document more resuable.

- Renamed files and directories to match Red Hat standards to simplify reuse.
- Consolidated duplicate SSO/Keycloak image files into the images directory

### CODE REVIEW

File: getting_started/topics/standalone/proc-installing-server-product.adoc [hunk 1]
```diff
@@ -9,27 +9,26 @@ For this sample instance of {project_name}, this procedure involves installation
...
 .Linux/Unix
 [source,bash,subs=+attributes]
 ----
-$ unzip rhsso-{project_version}.zip
+$ unzip rhsso-7.4.0.zip
 
 or
 
-$ tar -xvzf rh-sso-{project_version}.tar.gz
+$ tar -xvzf rh-sso-7.4.0.tar.gz
```
Comment (ASzc) [inline getting_started/topics/standalone/proc-installing-server-product.adoc]: Why replace the SSO version variable with a hardcoded version? That seems like it'd introduce more maintenance work.

Looking at document-attributes-product, I see project_version is `7.4.0.GA`, so maybe we need to add `project_versionFiles` as `7.4.0`?

**Review by ASzc — CHANGES_REQUESTED** [review]
Seems like all the content has been moved ok. I can't see anything that didn't make it into the post-reorg files somewhere.

One issue with the use of a hardcoded product version, see the comment.

**Comment by andymunro** [comment]
Thanks, @ASzc.  I actually had an attribute I was using for this situation in a different PR.  I have applied it here.  

Thanks for checking.   Looks like @pskopek will be able to get this into 7.4.2, which is great.

**Review by ASzc — DISMISSED** [review]
@abstractj LGTM

**Comment by abstractj** [comment]
@pskopek would you like to review, prior to merge?

**Review by pskopek — DISMISSED** [review]
LGTM!

**Comment by andymunro** [comment]
@pskopek @abstractj Thanks.  Is it okay to cherry-pick this for prod or do I need to wait till it is merged?

**Comment by andymunro** [comment]
@pskopek Please apply this PR first: https://github.com/keycloak/keycloak-documentation/pull/973

**Comment by abstractj** [comment]
@andymunro could you please resolve the conflicts so we can get it merged?

**Comment by andymunro** [comment]
@abstractj Can you check this again now.

**Review by pskopek — APPROVED** [review]
LGTM!

**Pull request merged**